PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/GfaReader.java

```diff
@@ -28,14 +29,26 @@ public GfaReader(String filename, int graphsize) {
     this.numNodes = graphsize;
     this.sc = new Scanner(GfaReader.class.getClassLoader().getResourceAsStream(filename));
     originalGraph = new OriginalGraph();
-    FullGfaReader.prepNodes(originalGraph, numNodes);
+    prepNodes(originalGraph, numNodes);
     read();
 
     sc.close();
     sc = null;
   }
 
   /**
+   * Initialize the nodes of a graph.
+   *
+   * @param graph the graph of which to initialize a set amount of node.
```

Comment:
Ok I will delete the file.
I actually only moved this code. It was first in the fullgfareader, but it isn't needed there anymore (it was not present in the GfaReader class because of code duplication issues) . The GfaReader class isn't used anymore, so I will delete it and rename the FullGfaReader class.